src/sentry/static/sentry/app/components/events/interfaces/threads/threadSelector/index.tsx

```diff
@@ -50,6 +50,13 @@ const ThreadSelector = ({threads, event, activeThread, onChange}: Props) => {
     };
   };
 
+  const getItems = () => {
+    const newThreads = [...threads];
+    const crashIndex = threads.findIndex(thread => !!thread?.crashed);
+    newThreads.splice(crashIndex, 1);
+    return [threads[crashIndex], ...newThreads].map(getDropDownItem);
+  };
```

Comment:
You might be interested in `lodash/partition`. We are using it in our codebase for this exact use case.

Reply:
Thank you for the tip! I've updated the code 😉